drivers/arm/i2c_master.h

```diff
@@ -61,6 +61,10 @@
 #    ifndef I2C1_DUTY_CYCLE
 #        define I2C1_DUTY_CYCLE STD_DUTY_CYCLE /* FAST_DUTY_CYCLE_2 */
 #    endif
+#    ifndef PAL_MODE_STM32_ALTERNATE_OPENDRAIN /* stm32f4 uses GPIOv2 */
+#        define I2C_PAL_MODE 4
```

Comment:
This kinda bypasses the `I2C1_SCL_PAL_MODE` stuff, and would prefer it was aligned in a separate `USE_GPIOV1` block like <https://github.com/qmk/qmk_firmware/blob/master/quantum/backlight/backlight_arm.c#L14>

Reply:
the pin mode definitions were changed, and also updated the i2c_master.c for this change.